README.md

```diff
@@ -2,38 +2,24 @@
 
 Onze blog staat live: http://geekbites.move4mobile.io
 
-
 #### How to install
-Allereerst even checken of je Ruby hebt geinstalleerd:
-
-`ruby -v`
-
-Als dat het geval is dan kun je volgende commando's uitvoeren:
-
-`sudo gem install bundler`
-
-`sudo gem install jekyll`
-
-`bundle`
-
-`jekyll serve`
-
-En je bent klaar voor het schrijven van je eigen Geekbite!
+First install Node.js and check which verion of Node you have.
+Eleventy requires Node-8 or newer 
 
-Mocht je toch tegen een probleem met dependencies aanlopen, dan kun je het volgende commando chain proberen:
+`node --version`
 
-`bundle update` 
+If that is the case you can run the following code to install the dependencies:
 
-`bundle install`
+`npm install`
 
-`bundle exec jekyll serve`
+When it found any vulnerabilities you can run the code  `npm audit fix` to fix them, or `npm audit` for details
```

Comment:
Dit deel hoeft er niet per se bij, is meer een algemeen 'npm handigheidje'

Of, je kan helemaal onderaan een # Tips sectie plaatsen, en 'em dan daaronder plaatsenn.